etc/greenboot.d/red/00_redboot_notification.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-set -euo pipefail
-
-echo "Health Check FAILURE! Boot status is RED"
```

Comment:
I wonder if this would still be useful (i.e. this would print in the journal right?)

Reply:
revive this as 00 then and make the motd update script 01?